src/converter/modules/blocks/create-article-metadata-topics.js

```diff
@@ -1,15 +1,16 @@
 import yaml from 'js-yaml';
 import { toBlock, newHtmlList } from '../utils/dom-utils.js';
+import { TOPICS } from '../blocks-translated-content.js';
 
-export default function createArticleMetaDataTopics(document, meta) {
+export default function createArticleMetaDataTopics(document, meta, reqLang) {
   const fullMetadata = yaml.load(meta);
   const metaElement = document.querySelector('.article-metadata');
   const feature = fullMetadata?.feature;
   // Article Metadata Topics
   if (feature && feature.trim() !== '') {
     const topicMetadata = feature.split(', ').map((item) => item.trim());
     const featureDivTag = document.createElement('div');
-    const items = ['Topics:'];
+    const items = [TOPICS[`${reqLang.replace('-', '_')}`]];
```

Comment:
languages in the URLs have a very specific format.

instead of doing `reqLang.replace('-', '_')}` just change the names in your `blocks-translated-content.js` to match the languages we support

Reply:
Done